public/jira/normalized/normalize.ts

```diff
@@ -111,7 +123,25 @@ type DefaultsToConfig<T> = {
   [K in keyof T as K extends `${infer FnName}Default` ? FnName : never]: T[K];
 };
 
-export type NormalizeConfig = DefaultsToConfig<typeof defaults>;
+export type NormalizeIssueConfig = DefaultsToConfig<typeof defaults>;
+export type NormalizeParentConfig = DefaultsToConfig<
+  Pick<typeof defaults, "getSummaryDefault" | "getHierarchyLevelDefault" | "getTypeDefault">
+>;
+
+export function normalizeParent(
+  issue: ParentIssue,
+  {
+    getSummary = defaults.getSummaryDefault,
+    getHierarchyLevel = defaults.getHierarchyLevelDefault,
+    getType = defaults.getTypeDefault,
+  }: Partial<NormalizeParentConfig> = {}
+) {
+  return {
+    summary: getSummary(issue),
+    hierarchyLevel: getHierarchyLevel(issue),
+    type: getType(issue),
```

Comment:
Is the status not needed as part of the normalized parent?

Reply:
Not currently. This function is based purely off the case Justin added a while back